content/20-vscode/_index.md

```diff
@@ -50,6 +50,16 @@ You can now test the extension by performing the following tasks.
     },
 ```
 
+{{% notice warning %}}
+If you are facing a challenge with extension with a pop-up error message like below:
```

Comment:
this sentence sounds a bit rough
"If you are facing a challenge with extension with a pop-up error message like below:"
I propose something like:
"If you get this pop-up error message, you might have to downgrade your ansible-core package as instructed:"

Reply:
updated, thanks!